src/read_agent_logging.in

```diff
@@ -19,23 +19,53 @@ URL="${1:-http://localhost:24231/metrics}"
 # Default to 10s if unset, as a fail-safe mechanism.
 INTERVAL=${COLLECTD_INTERVAL:-10}
 
+# 1. If the Logging Agent uses REST.
 # The part of the input that sed looks at looks like:
-#   stackdriver_successful_requests_count{grpc="false"} 255
-#   stackdriver_failed_requests_count{grpc="false",code="400"} 14
+#   stackdriver_successful_requests_count{grpc="false",code="200"} 255
+#   stackdriver_failed_requests_count{grpc="false",code="401"} 25
 #   stackdriver_failed_requests_count{grpc="false",code="404"} 18
-#   stackdriver_ingested_entries_count 7050
-#   stackdriver_dropped_entries_count 25
-#   stackdriver_retried_entries_count{code="500"} 2
-#   stackdriver_retried_entries_count{code="501"} 3
-#
+#   stackdriver_failed_requests_count{grpc="false",code="500"} 23
```

Comment:
When would this be possible? When the retry limit has been reached?

Reply:
That doesn't seem right. Retried requests will (hopefully) eventually succeed -- we should not be counting them as failed. The only reason to count them as failed is if we're no longer able to retry.

Reply:
Changed. Also created b/69976590 to fix that in the Logging Agent.

Reply:
Sigh... It looks like when the retry limit is reached, the output plugin just doesn't get invoked, so the buffer data will be discarded silently. We may have to keep track of the retry counter ourselves, and on the last retry increment the "dropped" entry count instead. Won't hold up this PR for that, but let's note that in b/69976590 so we can follow up.